packages/@lwc/integration-not-karma/test/component/feature-flag/index.spec.js

```diff
@@ -0,0 +1,44 @@
+import { createElement } from 'lwc';
+
+import Enabled from 'x/enabled';
+import Disabled from 'x/disabled';
+import NoFlag from 'x/no-flag';
+
+describe('Component feature flag enforcement', () => {
+    beforeEach(() => {
+        // Clean up after each test
+        document.body.innerHTML = '';
+    });
+
+    afterEach(() => {
+        // Clean up after each test
+        document.body.innerHTML = '';
+    });
```

Comment:
```suggestion
    afterEach(resetDOM);
```
A helper exists for this. 🙂